### DISCUSSION TITLE
Variables needed in the cateogrical method

### DISCUSSION THREAD

**Discussion by maude-v** [CREWdecisions/elicitr#61] [Q&A]
Hi everyone,
To start working on the categorical method, we need to check what parameters we need to be able to implement in the functions.
Here is the basis that we discussed, @canessas if you think of other parameters can you tell us? Thanks!

#### topics (third layer, i.e. mechanisms in our manuscript):
- there can be from 1 to infinite number of topics
- default names = topic 1, topic 2, topic 3, ...
- can be named by the user

#### sites (second layer):
- there can be from 1 to infinite number of sites
- the number or name of the sites does not have to be the same for every topic
- default names = site 1, site 2, site 3, ...
- can be named by the user

#### levels (first layer): 
- from 2 to infinite number of levels, default is 5. 
- the number or name of the levels does not have to be the same for every topic. Does it have to be same for every site inside a topic? (i would say yes)
- default names = if 5 levels: c("minimal", "minor", "moderate", "major", "massive"). if less or more levels: c(1,2,3,4,...)
- can be named by the user

#### experts
- there can be from 1 to infinite number of experts 
- the number or name of the experts does not have to be the same for every topic. Does it have to be same for every site inside a topic? 
#### estimates (first layer): 
- probabilities (0-1, must sum to one for all the levels)
- confidence (0-100, given in percent)

#### votes
- for bootstrapping method, the number of times you want to multiply the weight of experts by

#### other potential ideas
- turn on or of dirichlet (i.e. use dirichlet priors or not)
- allow users to apply only one method (chose either basic method or bootstrapping method instead of both together)

**Comment by sgvignali** [comment]
@maude-v Should we not better name "mechanisms" instead of "topics" to be consistent with the manuscript?

**Reply by maude-v** [reply]
Yes we can, I thought topic was more general, but if we're keeping sites anyway we might as well go with mechanisms too so we can justify the names 👍
